apps/idos-data-dashboard/src/routes/dashboard/credentials/components/grants-center.tsx

```diff
@@ -85,17 +85,20 @@ const Shares = ({ grants }: { credentialId: string; grants: idOSGrant[] }) => {
                   <Text>{grant.lockedUntil ? bigintToDate(BigInt(grant.lockedUntil)) : "-"}</Text>
                 </Td>
                 <Td isNumeric>
-                  <Button
-                    size="sm"
-                    variant="outline"
-                    colorScheme="red"
-                    isLoading={
-                      revokeGrant.isPending && revokeGrant.variables?.dataId === grant.dataId
-                    }
-                    onClick={() => onRevoke(grant)}
-                  >
-                    Revoke
-                  </Button>
+                  {grant.lockedUntil &&
+                  BigInt(grant.lockedUntil) <= BigInt(Math.floor(Date.now() / 1000)) ? (
+                    <Button
+                      size="sm"
+                      variant="outline"
+                      colorScheme="red"
+                      isLoading={
+                        revokeGrant.isPending && revokeGrant.variables?.dataId === grant.dataId
+                      }
+                      onClick={() => onRevoke(grant)}
+                    >
+                      Revoke
+                    </Button>
+                  ) : null}
```

Comment:
Can we display something like `Timelocked until $HUMAN_READABLE_TIME`?

Happy to punt to another PR.